Hai secret no more (#4952)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

				`	"No, I'm not interested."`
					decline

# Must be after the Quarg and Korath intro missions:
mission "ZZ Hai Rescue [A03] Microtransceiver Identification"
	landing
	name "Missing Researchers"
	description "Fly to <destination> to talk to Tarazed Shipyards about the microtransceivers."
	source "Kuwaru Efreti"
	destination "Wayfarer"
	to offer
		has "Hai Rescue [A02] Microtransceiver Identification: done"
	on offer
		log `The Korath identified the microtransceivers as using the same materials and design principles as their own technology, but extremely primitive and poorly-made. The Quarg provided a data card that Tarazed Shipyards may be able to use.`
		conversation
			`After you land, a hovering gray platform lowers to the entrance of your ship. You step on, and it raises you to a higher level of the ringworld where many Quarg and a few Korath are present. You find the first Quarg who will talk to you and show it the vial of microtransceivers.`
			`	The Quarg makes a series of wheezing sounds which approximate a laugh. "I am merely a pilot. I know nothing of microtransceivers. That is the technology of the Korath, and I will take you to one who is knowledgeable."`

> 			`	You pass through another two grey platform transports and reach a warm cave-like structure with a group of Korath and Quarg. From the holograms and equipment, it appears they're discussing science. One hologram looks like a blood cells attached to a tiny machine; this may be a microscale medicine laboratory.`

Author: ```suggestion
			`	You pass through another two gray platform transports and reach a warm cave-like structure with a group of Korath and Quarg. From the holograms and equipment, it appears they're discussing science. One hologram looks like a blood cells attached to a tiny machine; this may be a microscale medicine laboratory.`
```

			`	"Officially, the Syndicate maintains that they wiped Base Principles out to the last individual, but evidence never proved it had been conclusively terminated.`
			`	"For the demonic symbol, we'll have to go to another library in room 4196A21."`
			choice
				`	(Take the elevator to the 419th floor.)`
			`	When you arrive, Giti tells you, "This is a religious materials archive. Give me a minute here..." Her voice trails off as she starts typing on a terminal. She walks back and forth between several bookshelves skimming books for a few minutes.`
			`	"I found it," she says. "This is a modified version of the Symbol of Baphomet, the official symbol of an older incarnation of the Church of Satan. Contrary to popular belief at the time, they were never devil-worshipers of any sort. Instead, they believed in a sort of rationalist individualism and were one of the first organizations to fund large-scale settlements on Freedom. Since then, the symbol has been co-opted by miscreants.`

> 			`	"But you didn't come here for a theological history disquisition. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!`

Reviewer: you forgot the " after degree!

Author: No, when the next paragraph is the same person continuing to speak it's convention to leave the closing mark off, because it's not actually closing the spoken part.

Reviewer: I'm not sure that's the case in all of your conversations though

Reviewer 2: Hopefully should be. I check that when proofreading, but certainly possible I missed one!

Author: Yeah I generally trust W1zrad but if you spot one that's been missed, obviously suggest a correction.

> 			`	"The circle around the symbol looks almost like a wormhole. A red wormhole. I heard about one in Terminus near The Deep. Apparently nobody is able to enter one without their ship being torn to pieces. This symbol suggests there's another one in the middle of Syndicate space.`

Reviewer: same, forgot " after space.

			`	"This isn't a base; it's a colony."`
			`	You notice capital ships heading your way and about two dozen more preparing to leave the planet. It's time to go.`
	on complete
		payment 775000
	npc
		government "Pirate (Devil-Run Gang)"
		system "Devil-Hide"
		fleet "Devil-Run Warships" 2
		personality nemesis harvests plunders disables

mission "Hai Rescue [A09] Slaver Ships"
	landing
	name "Missing Researchers"
	description "Report back to Teeneep on <destination>. Payment is <payment>."
	source "Earth"
	destination "Darkwaste"
	to offer
		has "Hai Rescue [A08] Scanner: done"
	on offer
		conversation
			`The Earth authorities give you an unusual landing vector that leads to a secret docking bay below ground in Argentina. You land, and a Navy cadet meets you at your ship with an urgent message: "Republic Intelligence and a Navy Marshal want to see you as soon as possible. They have news about your microtransceiver transmissions. Please follow me."`
			`	Giti adds, "I'll take the scanners and data to some researchers and let you know when I have more information. Please go with the cadet. It sounds like you'll finally get some answers."`

> 			`	The cadet takes you to an elevator just outside your ship, down more than three hundred floors to a large meeting room. Within, you see two men and a woman in formal dress. One man has a Republic Intelligence uniform with the name "Agent Kopkow". Another wears a Navy full dress uniform that you don't immediately recognize. After a moment of thought, you realize he is a Marshal, the second-highest rank in the Navy, answering only to the Grand Marshal. His uniform is is labeled "Adurath Kaine". The woman is a Navy captain wearing the prominent Parliamentary Medal of Valor. Her name label says "Captain Anna Wallace."`

Reviewer 3: ```suggestion
			`	The cadet takes you to an elevator just outside your ship, down more than three hundred floors to a large meeting room. Within, you see two men and a woman in formal dress. One man has a Republic Intelligence uniform with the name "Agent Kopkow". Another wears a Navy full dress uniform that you don't immediately recognize. After a moment of thought, you realize he is a Marshal, the second-highest rank in the Navy, answering only to the Grand Marshal. His uniform is labeled "Adurath Kaine". The woman is a Navy captain wearing the prominent Parliamentary Medal of Valor. Her name label says "Captain Anna Wallace."`
```

			`	"Please, return to your ship and find the victims!"`
			choice
				`	(Return to your ship.)`
			`	You plug the data card into your computer, and a new "government" shows up: Smuggler. The ships will be colored differently than pirates.`
			`	It may be risky, but you should disable one and board it.`
			`	Teeneep gave you the name of a contact within a historical archive on <destination>. She wants you to speak to a woman named, "Giti Gilani" if you find one of the smuggling ships. Giti specializes in history of ship designs and space-faring groups, including the Hai.`
				accept

mission "Hai Rescue [A05] Boarded Smuggler"
	name "Missing Researchers"
	description "You have the information Giti Gilani will need. It is time to visit her on <destination>."
	boarding
	destination Earth
	to offer
		has "Hai Rescue [A04] Tarazed Analysis: active"
	source
		government "Smuggler (Hai Trafficker)"
	on offer
		conversation

> 			`As you approach the airlock of the smuggler ship <origin>, the door opens and a Hai rushes through pointing a gun back towards a human in the ship. The human and Hai shoot at the same time, killing each other. In his death throes, the Hai manages to toss a keycard in your direction. It is damaged beyond usefulness, but you can see a pair of logos on it. One looks like an angry cat in a circle and the other looks demonic.`

Reviewer 4: An angry black cat inside a circle is an anarcho-syndicalist symbol, so it fits for Base Principles (the original incarnation of the organisation under that name being well-educated, highly connected anarcho-syndicalists).

				`	(Ruminate first.)`
			`	You've had few dealings with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle. Murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament.`
			`	There are only five Navy Marshals, and one Grand Marshal; this man's high rank means he reports parliament regularly. People this high in the navy tend to deal more with political issues, rather than combat or strategy, so you wonder why they picked him to talk to you.`
			`	The Parliamentary Medal of Valor is only given to one person per year; this captain must have done something especially heroic. She must also have supporters in Parliament to receive the award in the first place. That will give her some freedom to act independently, like Danforth does.`
			`	The three of them are looking at you confused, wondering why you've stopped and stared at them for a half minute.`
			choice
				`	(Stop staring blankly and speak to them.)`
			label join

> 			`	Marshal Kaine introduces himself, "I'm the Marshal of the Third Armada, the guards of the Far North and the uninhabited regions near it. I know you've met one of my admirals, a good man albeit one who flouts the rules a bit too much." He gives a stern look at Captain Wallace. "Some of my carriers have detected the signals you've found. Actually, we see them quite regularly in the ships of known slave runners. The runners don't always carry slaves; they do other smuggling missions as well."`

Reviewer 5: If you want to take a pass at this, please do so. Otherwise I'll work on it after I fix the mission coding bugs.

> 			`	Marshal Kaine introduces himself, "I'm the Marshal of the Third Armada, the guards of the Far North and the uninhabited regions near it. I know you've met one of my admirals, a good man albeit one who flouts the rules a bit too much." He gives a stern look at Captain Wallace. "Some of my carriers have detected the signals you've found. Actually, we see them quite regularly in the ships of known slave runners. The runners don't always carry slaves; they do other smuggling missions as well."`

Author: I have made a simple change.

				`	(Come back later.)`
					defer
			`	You walk twenty meters into the cave and pass through an open doorway. There's a variety of medical machinery and some computers in a makeshift base. Some of the people you've rescued are present, with some more humans and Hai you haven't met. A few of them have pulse rifles. A woman walks towards you.`
			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a civil rights advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and humans victims are here; they were rescued by other kind merchants in human space."`
			`	"My organization researched their problem and discovered some patterns to the disappearances. We've found these kidnappings have been going on for several years. What is strangest is the microtransceivers in most of the kidnapped victims."`
			choice
				`	"What patterns have you noticed in the kidnappings?"`
					goto patterns
				`	"Tell me about the microtransceivers."`
					goto microtransceivers
				`	"How long have the kidnappings been happening?"`
					goto kidnappings
			label questions
			`	"Do you have any other questions?"`

> 			choice

Author: Probably, I'll leave this open for later when I maybe feel like expanding on bits.

				`	"How do you know all of this?"`
					goto know
				`	"Why are you on an uninhabited planet?"`
					goto uninhabited
				`	"Is anyone helping you?"`
					goto anyone
				`	"What patterns have you noticed in the kidnappings?"`
					goto patterns
				`	"Tell me about the microtransceivers."`
					goto microtransceivers
				`	"How long have the kidnappings been happening?"`
					goto kidnappings
				`	"How can I help you?"`
					goto help
			label uninhabited

> 			`	Teeneep looks at you as if you were daft, "The kidnappings happen on Hai and human worlds. The kidnappers have no reason to search an uninhabited planet for kidnapping victims. This gives us a safe place to hide any critical people or information."`

Author: Is the range of the transceivers discussed later? This might be a plothole I need to close.

> 			`	Teeneep looks at you as if you were daft, "The kidnappings happen on Hai and human worlds. The kidnappers have no reason to search an uninhabited planet for kidnapping victims. This gives us a safe place to hide any critical people or information."`

Reviewer 6: There's some rather confusing discussion later on that I'll review in context when I get to it. (Unfortunately I haven't simultaneously had both the time and the energy to do the next batch of reviewing recently.) In this conversation we're told Teeneep's group can detect them within 30 metres, but also that they seem to broadcast through hyperspace. From the kidnappers' perspective, there doesn't seem to be much point to tagging victims with a complicated system that has a range shorter than the length of most ships, so I would expect the range to be rather longer than that if suitable equipment is used.

				goto questions
			label microtransceivers
			`	A Hai in a medical uniform explains, "The victims each have have thousands of microscale transmitter-receivers in their body, each the size of a single blood cell. We call them microtransceivers. They act together as one big transmitter and receiver, sending and receiving directional messages over great distances."`

> 			`	"This technology is clearly not of human or Hai origin. It has some similarities to some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding, but the construction of these  is significantly beyond our science and some materials involved have properties unlike anything we've ever studied up close. Another race's hand has been involved in this technology, a race that has mastered miniaturized technology and is comfortable with its principles."`

Author: I believe we should use a semicolon instead of a "-", just for style purposes.

> 			`	"This technology is clearly not of human or Hai origin. It has some similarities to some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding, but the construction of these  is significantly beyond our science and some materials involved have properties unlike anything we've ever studied up close. Another race's hand has been involved in this technology, a race that has mastered miniaturized technology and is comfortable with its principles."`

Reviewer 2: I'm down, but we'd need to do a little more editing to make that work (the phrase after a semicolon needs to be a complete sentence).
```suggestion
			`	"This technology is clearly not of human or Hai origin. It has similarities to some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding, but the construction of these is significantly beyond our science, and some materials involved have properties unlike anything we've ever studied up close. Another race's hand has been involved in this technology; we're looking for one that has mastered miniaturized technology and is comfortable with its principles."
```
This is the best I can think of off the top of my head. I can't figure out a way to preserve the `a race` bit and still make it a complete sentence, which is unfortunate because that bit's cool. Thoughts?

			`	"We've spent the last two years researching this problem, but it helped a lot to meet the people who have been rescued."`
				goto questions
			label anyone
			`	A Hai near Teeneep responds. "Yes... and no. Apart from our own people, we're regularly in contact with police. They investigate the disappearances, and sometimes find the missing person. Other times, they eventually have to give up and work on other crimes."`
			`	"Our biggest problem is the lack of help in human space. Local police are generally overwhelmed with crime in the regions where the kidnapping victims are found. There, kidnappings are commonplace. In the areas where the victims were taken, there's virtually no evidence. Your navies and intelligence agencies haven't the time to help us; they're busy cleaning up after the war. There is a lot of unfinished business in Syndicate space they don't want to talk about."`

> 			`	"Since this problem is tiny compared to the total crime in Hai space, the Hai Elders haven't made it a large priority. They tell me that, if we can prove the kidnappings are from a coordinated effort, they'll put all the resources they can spare into stopping the crime. So far, they say, their investigators have been unable to discover anything."`

Author: What if we simply went with something along the lines of "while the crime in Hai space is rather more manageable, the same lack of evidence makes it hard for the Elders to justify a dedicated task force when more visible priorities still dominate the news feeds." Thoughts?

> 			`	"Since this problem is tiny compared to the total crime in Hai space, the Hai Elders haven't made it a large priority. They tell me that, if we can prove the kidnappings are from a coordinated effort, they'll put all the resources they can spare into stopping the crime. So far, they say, their investigators have been unable to discover anything."`

Reviewer 2: I like it! Add it like this?
```suggestion
			`	"While the crime in Hai space is rather more manageable, the same lack of evidence makes it hard for the Elders to justify a dedicated task force when more visible priorities still dominate the news feeds. They tell me that, if we can prove the kidnappings are from a coordinated effort, they'll put all the resources they can spare into stopping the crime. So far, they say, their investigators have been unable to discover anything."`
```

			`	The Quarg makes a series of wheezing sounds which approximate a laugh. "I am merely a pilot. I know nothing of microtransceivers. That is the technology of the Korath, and I will take you to one who is knowledgeable."`
			`	You pass through another two grey platform transports and reach a warm cave-like structure with a group of Korath and Quarg. From the holograms and equipment, it appears they're discussing science. One hologram looks like a blood cells attached to a tiny machine; this may be a microscale medicine laboratory.`
			`	The Quarg and Korath discuss your microtransceivers for a few minutes, placing them in various instruments.`
			`	Finally you have your answer. A Quarg scientist tells you, "These appear to be Korath in origin, but makeshift and primitive. Even the Korath Exiles are more advanced. Those who made these have little resources and simple machinery. This will tell you how to detect these microtransceivers," it hands you a standard data card compatible with human computers.`

> 			`	"Take this to Tarazed Shipyards."`

Author: I always figured that the Exiles, being a smaller group that was more attached to their technology, likely had a higher proportion of scientists and the like, and put the effort in to make sure they absconded with everything they could. Whether that's data, schematics, prototypes, etc.
Whereas the Efreti are really in an ongoing humanitarian crisis, and practically speaking, even if they have those things, they've given up some stuff, and a lot of other things are likely buried under the weight of administration.

> 			`	Finally you have your answer. A Quarg scientist tells you, "These appear to be Korath in origin, but makeshift and primitive. Even the Korath Exiles are more advanced. Those who made these have little resources and simple machinery. This will tell you how to detect these microtransceivers," it hands you a standard data card compatible with human computers.`
> 			`	"Take this to Tarazed Shipyards."`

Reviewer 2: That's reasonable. Here's the suggestion with my other edits but the sentence added back in:
```suggestion
			`	Finally, you have your answer. A Quarg scientist tells you, "These appear to be Korath in origin, but makeshift and primitive. Even the Korath Exiles are more advanced. Those who made these have little resources and simple machinery. This will tell you how to detect these microtransceivers," it explains, handing you a standard data card compatible with human computers. "Take this to Tarazed Shipyards."`
```

			`	After a series of shuttles and trains, you reach his house and knock on the door.`
			`	He's a little groggy, but manages to ask, "Yes?"`
			`	You explain about the microtransceivers, the Quarg, and the kidnappings. You avoid mentioning that some of the people are Hai, just in case he's never heard of them.`
			`	He seems to awaken immediately, "You found a lead on my wife's disappearance? Yes, I can help you modify your ship's communicator to detect these signals. Let's go to my laboratory at once."`
			choice
				`	(Head to the laboratory.)`
			`	After another series of shuttles and trains, you reach what looks like a spaceport. From the extra equipment and unusual ship designs, you realize it is a research shipyard. Goraz's laboratory is an entire docking bay with a modified Blackbird. He takes you on board and walks you to the communications room.`
			`	"This is a non-standard communicator, but it should be easy to reprogram yours." He takes the data card you got from the Quarg and plugs it into the communicator. After several minutes of typing, he speaks again.`

> 			`	"There is nothing in this vicinity, but I was able to patch into our planet-scale receivers. They are extremely sensitive, and connected to the communication relay networks throughout human space. Those communicators aren't designed for precise tasks like this, but I can tell you where to look."`

Author: I read it as more a matter of, the comms network is capable of picking up on any unprotected general-broadcast signal. (which these are, though weak). It's not scanning for them, simply now bring told to look through what it's receiving for something it wasn't considering significant before.
Whereas in war you wouldn't be necessarily broadcasting your ship and position on unencrypted channels etc.

> 			`	"There is nothing in this vicinity, but I was able to patch into our planet-scale receivers. They are extremely sensitive, and connected to the communication relay networks throughout human space. Those communicators aren't designed for precise tasks like this, but I can tell you where to look."`

Reviewer 6: It's taken me far too many attempts to write this in a way that doesn't involve pages of excessive technical detail, so it'll probably look like a bunch of unjustified assertions. I'm happy to expand on any of them if necessary, though.

The purpose of a communications network is primarily to provide efficient communications from any point in the network to any other point. The only practical way to do that is to use a hierarchical architecture with systems that deal with high-level and network-wide concerns (routing, billing, interconnection with other networks, etc.) at the top and systems that deal with low-level and very local concerns (RF signalling, channel allocation, managing interference, etc.) at the bottom. The whole system is aggressively optimised for moving comms data from one part of the network to another, with maybe a sliver of capacity left over for things like monitoring and control of the network as a whole.

As a result, someone sitting in the network operations centre only really deals with the high-level stuff and whatever predefined monitoring and control has been set up across the network as a whole. They could probably tell you that there are a surprising number of calls between Vladivostok and Nairobi today. They almost certainly couldn't tell you that all the terrestrial TV stations in Jakarta have stopped broadcasting, even though the hardware in cellphone base stations is technically capable of detecting that.

With sufficiently futuristic (or even modern) equipment, it may be possible to reconfigure base stations across the entire network to detect and report on arbitrary signals -- this is pretty much what I expect Goraz is doing -- but doing that would require diverting capacity from the core business of providing a communications service. At the very least, this would be immediately apparent to everyone looking at the system status. Most likely, there would be alarms going off and people being pulled out of bed and dropped calls all across the galaxy and hastily convened disciplinary proceedings (probably ending in termination, possibly including criminal charges) and generally a bad time all around.

(If you can do this sort of thing *without* setting off lots of alarms, all of your security and infrastructure personnel need to be fired immediately. Out of torpedo tubes. Also, you're probably so overprovisioned that you're already being outcompeted by service providers who are running more efficient networks.)

Finally: If standard communications gear can detect signals that the Hai can't see beyond 30m (even with instructions from the Quarg), it can definitely detect the waste emissions of an uncloaked Navy fleet.

> 			`	"There is nothing in this vicinity, but I was able to patch into our planet-scale receivers. They are extremely sensitive, and connected to the communication relay networks throughout human space. Those communicators aren't designed for precise tasks like this, but I can tell you where to look."`

Reviewer 6: Despite all the above, I think this part of the story is fundamentally sound and with a little tweaking it could be a whole lot better. Here are some ideas:

 * We've already established that Goraz has a personal interest in the kidnappings, and he also has access to significant signal-processing capabilities. Maybe he's made contact with a bunch of other people who've had similar experiences (perhaps through Teeneep's group), and together they've been collecting broad-spectrum scans of all ships in the general vicinity of the known kidnappings. Goraz has been trying (for months, years, however long his wife has been missing) to find some pattern in the data that would identify the kidnappers. The microtransceiver information you provide lets him run a much more targeted analysis and pinpoint systems (or maybe even individual ships) in his existing dataset without having to use a galactic comms network. *This lets Goraz provide the same information he currently provides, but without the realtime surveillance issues.*
 * Maybe Goraz doesn't have a network of contacts or a bunch of data, or maybe he's just not that obsessed. Since we already know where we found the escaped Hai victims, Goraz can modify our gear and we could go back there to look for kidnappers. *This makes Goraz a little less helpful, but it links back to stuff we've already done and reinforces the continuity of the story as a whole.*
 * Not really directly related, but I found the kidnapper ships incredibly difficult to find and attack before they jumped out of the system (possibly exacerbated by flying an overstuffed Korath Dredger with a turning circle measured in parsecs). Maybe it would be interesting to have those show up as merchants (like the disguised pirate bounty missions) and have the player scan them to discover whether they're kidnappers or not. *Not sure if this is actually possible, but I think it would be a lot more fun than trying to hit a ship that frequently jumps away just as you target it.*

> 			`	"There is nothing in this vicinity, but I was able to patch into our planet-scale receivers. They are extremely sensitive, and connected to the communication relay networks throughout human space. Those communicators aren't designed for precise tasks like this, but I can tell you where to look."`

Author: Thankyou for the detailed response. Neither Nomadic nor I are comms people by any stretch. I will see what I can do with this section to align things to your ideas.

> 			`	"There is nothing in this vicinity, but I was able to patch into our planet-scale receivers. They are extremely sensitive, and connected to the communication relay networks throughout human space. Those communicators aren't designed for precise tasks like this, but I can tell you where to look."`

Reviewer 3: One of those first two bullet points above still seems like the way to go to me.

> 			`	"There is nothing in this vicinity, but I was able to patch into our planet-scale receivers. They are extremely sensitive, and connected to the communication relay networks throughout human space. Those communicators aren't designed for precise tasks like this, but I can tell you where to look."`

Author: I believe I have now reworked this in line with a combo of those two points.

			`	He gives you back your data card, and second one. "This new card will reprogram your communicator to find the ships. They'll show up as, 'Smuggler' on your display. Search the uninhabited areas between the Far North and Syndicate space, plus any nearby pirate systems."`
			`	"Please, return to your ship and find the victims!"`
			choice
				`	(Return to your ship.)`
			`	You plug the data card into your computer, and a new "government" shows up: Smuggler. The ships will be colored differently than pirates.`
			`	It may be risky, but you should disable one and board it.`

> 			`	Teeneep gave you the name of a contact within a historical archive on <destination>. She wants you to speak to a woman named, "Giti Gilani" if you find one of the smuggling ships. Giti specializes in history of ship designs and space-faring groups, including the Hai.`

Author: This is now in contrast where you left them in place with the guy who could modify your communicator.

> 			`	Teeneep gave you the name of a contact within a historical archive on <destination>. She wants you to speak to a woman named, "Giti Gilani" if you find one of the smuggling ships. Giti specializes in history of ship designs and space-faring groups, including the Hai.`

Reviewer 2: My bad (did chunks of this over a few separate days). That suggestion should be updated now to include that fix.

	destination Earth
	to offer
		has "Hai Rescue [A04] Tarazed Analysis: active"
	source
		government "Smuggler (Hai Trafficker)"
	on offer
		conversation
			`As you approach the airlock of the smuggler ship <origin>, the door opens and a Hai rushes through pointing a gun back towards a human in the ship. The human and Hai shoot at the same time, killing each other. In his death throes, the Hai manages to toss a keycard in your direction. It is damaged beyond usefulness, but you can see a pair of logos on it. One looks like an angry cat in a circle and the other looks demonic.`
			`	Soon after, you hear explosions in the ship and see an escape pod float away. It seems the smugglers set the self-destruct and signaled for help. The pod has no microtransceiver transmissions, so it must be one of the smugglers. Apparently the smugglers care more about not being caught than about finishing their mission.`
			`	Now it's time to set a course for <destination> where you can finally get some answers.`
			`	Multiple hyperspace windows begin to form. You prepare for battle.`
				launch
	npc
		government "Pirate"
		personality nemesis harvests plunders entering
		fleet "pirate raid" 3

> mission "Hai Rescue [A06A] Smuggler Ambush"

Author: So far no one who's playtested it has raised any confusion. Some have said it's challenging but I'm reasonably confident at this stage that in-game it's clear enough. Also only the first one is technically required to progress. The others are just there for flavour.

	on enter "Deep Space 7C"
		log `Passed through the wormhole in Syndicate space to a region just outside the Milky Way galaxy. It looks like a good place to hide.`
		dialog `It takes your computer a few seconds to triangulate, but it says you've left the Milky Way galaxy and are now in intergalactic space. Your computer is identifying the system name as, "Deep Space 7C" from old astronomical surveys done in the late 21st century. This is definitely a good place to hide. You should return to Giti now and see what more she's found out.`
	on offer
		log "People" "Giti Gilani" `Giti is a research librarian on Earth that specializes in archaic space flight knowledge.`
		log "Factions" "Base Principles" `A clandestine organisation that existed within the Syndicate that was founded upon anarcho-syndicalist ideals. Its ranks eventually became bloated by enthusiastic recruits from Freedom that were keen to get rich on top of their freedeom, and that were more anarchist than anarcho-syndicalist. The resulting tensions undermined the group and Syndicate Internal Affairs scoured them from the organization.`

> 		log "Factions" "Syndicate Internal Affairs" `An internal police force within the Syndicate that enforces the Syndicate's laws.`

Author: Log Factions don't necessarily have explicit governments in game as far as I know. It's simply added more as an organisation that the player is now explicitly aware of.

		log "Giti identified the angry cat symbol. It is from a long-defunct subversive group within the Syndicate called Base Principles. Somehow they obtained alien technology, and used the knowledge gained from it to advance their influence within the organization. A half-century ago, Syndicate Internal Affairs scoured them from the Syndicate's ranks in an extremely thorough and prejudiced inquisition."
		conversation
			`After landing, you search for the "Republic Historical Space-Flight Museum," and find there are seventeen museums by that name on Earth.  Only one has a research library, so you contact the library. Giti Gilani no longer works there; she's moved to a library in the Navy Intelligence Observatory in Moscow.`
			`	The Navy observatory was easier to find; when they hear your name, they put you in contact with Giti immediately. She is excited to see the two symbols you found and asks you to visit her. It takes three shuttle journeys and one high-speed rail trip, but you reach the building in under three hours.`
			choice
				`	(Enter the building.)`

> 			`	You're ushered to an elevator and given a room number: 3017A19, the 301st floor, section 7A, room 19. It's amazing how large Earth's megabuildings have become. Some planets have lower population than one of these buildings.`

Reviewer 2: ```suggestion
			`	You're ushered to an elevator and given a room number: 3017A19 - the 301st floor, section 7A, room 19. It's amazing how large Earth's megabuildings have become. Some planets have fewer populations than one of these buildings.`
```

Author: Now that you've drawn my attention to it, I actually think "Some whole planets have less population than one of these buildings." would be better.

			`	Giti is even more excited in-person than she was by holocommunicator. She's already researched the Syndicate-like symbol, but needed more detail about the demonic one. You also show her the information about the microtransceivers.`

> 			`	"This cat symbol was used by a clandestine organization that worked within the Syndicate over a half-century ago. They called themselves 'Base Principles.' They were anarcho-syndicalists infiltrating the Syndicate from within, supposedly advancing their positions through research findings gleaned from captured alien equipment, but nobody knows where they got it. The organization was eventually torn in two by a collection of regular anarchists and other unsavory characters looking to profit off the backdoor access to the Syndicate; this group kidnapped researchers from other institutes to expand their capabilities."`

Reviewer 2: ```suggestion
			`	"This cat symbol was used by a clandestine organization that worked within the Syndicate over a half-century ago. They called themselves 'Base Principles.' They were anarcho-syndicalists infiltrating the Syndicate from within, supposedly advancing their positions through research findings gleaned from captured alien equipment, but nobody knows where they got it. The organization was eventually torn in two by a collection of regular anarchists and other unsavory characters looking to profit off the backdoor access to the Syndicate; this group kidnapped researchers from other institutes to expand their capabilities.`
```
Mentioning it here - this applies to all the spots I'm adding suggestions for below: each paragraph that continues dialogue from the same person shouldn't have an ending quotation mark. Looks like something Nomadic didn't know about/missed?

> 			`	"These others didn't cover their tracks well enough, the trail led back to the syndicalists that started it all in the first place, and Syndicate Internal Affairs put a stop to the organization with extreme prejudice. Most of the anarchist leaders were killed in battle or executed after capture, the internal players simply... disappeared."`

Reviewer 2: ```suggestion
			`	"These others didn't cover their tracks well enough, the trail led back to the syndicalists that started it all in the first place, and Syndicate Internal Affairs put a stop to the organization with extreme prejudice. Most of the anarchist leaders were killed in battle or executed after capture, but the internal players simply... disappeared.`
```

> 			`	"Officially the Syndicate maintains that they wiped Base Principles out to the last individual, but evidence never did prove it had been conclusively terminated."`

Reviewer 2: ```suggestion
			`	"Officially, the Syndicate maintains that they wiped Base Principles out to the last individual, but evidence never proved it had been conclusively terminated.`
```

> 			`	"For the demonic symbol, we'll have to go to another library in room 4196A21."`

Reviewer 2: ```suggestion
			`	"For the demonic symbol, we'll have to go to another library: room 4196A21."`
```

Author: As a spoken line, I'm not sure this feels totally right.

Reviewer 2: I think either's fine, but if you prefer the original I'm cool with that.

			choice
				`	(Take the elevator to the 419th floor.)`
			`	When you arrive, Giti tells you, "This is a religious materials archive. Give me a minute here..." Her voice trails off as she starts typing on a terminal. She walks back and forth between several bookshelves skimming books for a few minutes.`

> 			`	"I found it," she says. "You found a modified version of the Symbol of Baphomet, the official symbol of an older incarnation of the Church of Satan. They were one of the first organizations to fund large-scale settlements on Freedom, though the symbol was ultimately co-opted by miscreants. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!"`

Author: The bit about it being symbology that has since been co-opted by miscreants is important. I don't mind the expansion of the detail, but the fact that _it is now being abused_ is important here.

> 			`	"I found it," she says. "You found a modified version of the Symbol of Baphomet, the official symbol of an older incarnation of the Church of Satan. They were one of the first organizations to fund large-scale settlements on Freedom, though the symbol was ultimately co-opted by miscreants. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!"`

Reviewer 6: Yeah, that makes makes sense.

My intention with the suggested change was to show what I meant more clearly, not necessarily for it to be used verbatim. This is your and NV's story, and (obvious tyops aside) any changes I suggest are for you to use, modify, or reject as you see fit.

> 			`	"I found it," she says. "You found a modified version of the Symbol of Baphomet, the official symbol of an older incarnation of the Church of Satan. They were one of the first organizations to fund large-scale settlements on Freedom, though the symbol was ultimately co-opted by miscreants. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!"`

Reviewer 2: ```suggestion
			`	"I found it," she says. "This is a modified version of the Symbol of Baphomet, the official symbol of an older incarnation of the Church of Satan. Contrary to popular belief at the time, they were never devil-worshipers of any sort. Instead, they believed in a sort of rationalist individualism and were one of the first organizations to fund large-scale settlements on Freedom. Since then, the symbol has been co-opted by miscreants.`
			`	"But you didn't come here for a theological history disquisition. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!`
```
Can I suggest a combination? I've also added a couple grammar/style edits of my own here.

> 			`	"The circle around the symbol looks almost like a wormhole. A red wormhole. I heard about one in Terminus near The Deep. Apparently nobody is able to enter one without their ship being torn to pieces. This symbol suggests there's another one in the middle of Syndicate space."`

Reviewer 6: Using an Ember Waste wormhole hole raises some unanswered questions:
* All Ember Waste wormholes need to be stabilised with keystones, and all other non-Ember Waste wormholes don't. What makes this one different?
* If you have a jump drive and a keystone (both of which are apparently required) and are willing to explore strange region of space, it's kind of hard to not run into the Remnant.
* Even if you don't run into the Remnant, surely they have access to jump drives (perhaps only have a handful, like the Syndicate) and would probably have found this area of space at some point. If nothing else, an Ember Waste wormhole outside the Ember Waste would surely be something they'd want to take a closer look at.
* If this whole thing predates contact with the Hai (as implied by Teeneep earlier) and they don't have contact with the Remnant, where would they have acquired a keystone?

I think it would make more sense for this to be a normal wormhole, or require some new kind of key that would be more easily available in human space. Perhaps a "philosopher's stone"?

Author: You have 4 questions here:
1. Just because wormholes in the ember waste are unstable, does not mean that unstable wormholes may only exist in the ember waste.
2. & 3. I would like to use a system hazard to apply cloak disruption in one of the systems beyond it, thus acting as a stern discouragement for the Remnant to have been around here because they don't like meeting people.
4. The fact that the organisation may pre-date contact with the Hai does not necessarily mean that their current base of operations and general MO predates it. Plus, the Hai previously occupied this part of space. who's to say that there wasn't the odd keystone unearthed from times past that was accidentally left behind. ;) (This is something I would like to expand upon in a side-quest later, but it's not required for the main PR.)

Reviewer 6: All good points. Perhaps putting the Ember Waste hazard in the Devil-Run system to make it more of a disconnected section of the Waste (it's certainly close enough for that to work) would avoid questions and also make the area just that little bit less hospitable?

Author: This is waiting on #3599

> 			`	"The circle around the symbol looks almost like a wormhole. A red wormhole. I heard about one in Terminus near The Deep. Apparently nobody is able to enter one without their ship being torn to pieces. This symbol suggests there's another one in the middle of Syndicate space."`

Reviewer 2: ```suggestion
			`	"The circle around the symbol looks almost like a wormhole. A red wormhole. I heard about one in Terminus near The Deep. Apparently nobody is able to enter one without their ship being torn to pieces. This symbol suggests there's another one in the middle of Syndicate space.`
```

			`	"Fly through the wormhole at the coordinates on that card, if you can, come back and tell me what you see. In the meantime, I'll ask other researchers about the microtransceivers."`
			choice
				`	(Thank her and return to your ship.)`

> 			`	Upon returning to your ship, you ponder Giti's findings. The coordinates are for a system in Syndicate space that is not reachable by starlane, so you'll need a Jump Drive. The red wormhole requires another key.`

Reviewer 6: `starlane` is a new word that doesn't appear anywhere else in the game. By far the most common name for the things that hyperdrives use is `hyperspace link`, but there are also a handful of places where they're called `hyperspace connection`, `hyperlink`, or `hyperlane`.

Author: Good catch. This is one of the sections written by Nomadic before he departed so I do appreciate people keeping an eye out for deviations like this.

			`	Once you've passed through that wormhole, hopefully Giti will have more information for you.`
				accept
	on complete
		log `Giti asked to go to the unexplored region beyond the Syndicate wormhole to seek a base. She's waiting at the Earth spaceport with 7 tons of cargo.`
		conversation
			`After a long series of shuttles, high-speed rails, and a several-minute-long elevator ride, you reach Giti's office.`
			`	As soon as she sees you she asks, "What did you find out?" You explain your journey to her.`

> 			`	She responds, "Wait... I... lemme check..." followed by some typing on a nearby terminal. "Yes, you're right. In the late 21st century, there was an effort to map stars in that intergalactic region. They should not have been so visible, but they were due to some strange anomalies. The strangest of all was not a star; it was an object with unknown properties that confuse our scanners even today."`

Reviewer 2: `They should not have been so visible, but they were due to some strange anomalies`
This sentence, especially the second use of `they`, is a little confusing. Can you think of another way to phrase it? I'd suggest something myself, but I think I might butcher its meaning.

Author: ```suggestion
			`	She responds, "Wait... I... lemme check..." followed by some typing on a nearby terminal. "Yes, you're right. In the late 21st century, there was an effort to map stars in that intergalactic region. By all rights these stars should not have been so visible, but some strange anomaly made that part of the sky clearer for a while. The strangest thing we spotted was not a star; it was some kind of object with unknown properties that confused our scanners. Even today I'm not sure we'd make sense of it, but the anomaly passed and we couldn't even make an attempt now."`
```
Thoughts?

Reviewer 2: Just one comma missing (added below). But yeah, that helps a whole lot
```suggestion
			`	She responds, "Wait... I... lemme check..." followed by some typing on a nearby terminal. "Yes, you're right. In the late 21st century, there was an effort to map stars in that intergalactic region. By all rights these stars should not have been so visible, but some strange anomaly made that part of the sky clearer for a while. The strangest thing we spotted was not a star; it was some kind of object with unknown properties that confused our scanners. Even today I'm not sure we'd make sense of it, but the anomaly passed, and we couldn't even make an attempt now."`
```

			`	"If the Syndicate criminals and kidnappers are through the wormhole, that's the system they'll be in."`
			`	"Let me contact some people. Hold on..." Her voice trails off as she begins typing on a nearby terminal.`
			choice
				`	(Wait.)`
					goto wait
				`	(Interrupt her.)`

> 			`	"What? I... Aaa! Don't startle me like that. I just need a minute. Hold on..."`

Reviewer 2: This makes me smile. I love the little bits of flavor tossed in

			label wait
			`	"Okay. I've arranged for some advanced scanning equipment that can see past most types of radiation and a few hundred meters of solid rock. I'll have to come with you on this trip."`
			`	"Oh! I almost forgot to tell you the good news. I found some people who know about the microtransceiver transmissions. Navy carriers have sensitive communication arrays that can pick up the transmissions. The Navy is combing their communication logs and will have some answers when we return."`

> 			`	"Meet me in the spaceport when you have a bunk and 7 tons of cargo space free. Oh, and make sure you have a ramscoop. The navy will pay for this trip."`

Reviewer 2: ```suggestion
			`	"Meet me in the spaceport when you have a bunk and 7 tons of cargo space free. Oh, and make sure you have a ramscoop. The Navy will pay for this trip."`
```

mission "Hai Rescue [A08] Scanner"
	name "Missing Researchers"
	description "Return to <destination> after scanning the Devil-Hide system. Payment is <payment>."
	source "Earth"
	waypoint "Devil-Hide"
	passengers 1
	cargo "advanced scanning equipment" 7
	blocked `You need room for Giti and her <cargo>.`
	on visit
		dialog `Giti or her equipment are in one of your escorts. You need to wait for your escorts to arrive before landing again.`
	to offer
		has "Hai Rescue [A07] Logo Identification: done"
	on offer
		conversation

> 			`Giti enters your ship after some navy workers load scanning equipment. She's eager to see first-hand the intergalactic space she's only theorized about in the past.`

Reviewer 2: ```suggestion
			`Giti enters your ship after some Navy workers load scanning equipment. She's eager to see first-hand the intergalactic space she's only theorized about in the past.`
```

		conversation
			`Giti turns on the scanning equipment.`
			`	"Okay, let's do the planet first..." Her voice trails off. "Wow! That's odd. We'll need a scanner about ten thousand times more powerful to scan that planet."`
			`	 "I can see a bit of the surface though. The planet is odd in every way. It is completely inhospitable to all known forms of life. There's a huge network of caves and caverns coated with a crystal that is radiating... something the computer can't identify... and deuterium. The denizens of this system siphon the deuterium to make hyperspace fuel. Also, the planet has about a thousand times the mass it should, but somehow the gravity is around Earth normal."`
			`	"Now the moon... hmmm... okay. It's just an ordinary chunk of rock, but the planet's radiation is disturbing the sensors. I can only see a few dozen meters below ground."`

> 			`	"I found your missing researchers. They're in a facility beneath the moon's surface. There are three different species there: Hai, human, and some kind of reptile." You realize she probably means a Korath. "No Alphas though. I can only see the uppermost part of the facility. They have hydroponics, big cargo bays, docking bays for ships, and some spacious living areas. I detect 1,387 life signs spanning all age ranges, and that's just in the upper three floors of the facility."`

Reviewer 2: A little strange that Giti, who is obviously very eager to learn all about that stuff, doesn't really ask any questions about the reptiles/Korath. Minor thing though.

			`	"This isn't a base; it's a colony."`
			`	You notice capital ships heading your way and about two dozen more preparing to leave the planet. It's time to go.`
	on complete
		payment 775000
	npc
		government "Pirate (Devil-Run Gang)"
		system "Devil-Hide"
		fleet "Devil-Run Warships" 2
		personality nemesis harvests plunders disables

mission "Hai Rescue [A09] Slaver Ships"
	landing
	name "Missing Researchers"
	description "Report back to Teeneep on <destination>. Payment is <payment>."
	source "Earth"
	destination "Darkwaste"
	to offer
		has "Hai Rescue [A08] Scanner: done"
	on offer
		conversation

> 			`The Earth authorities give you an unusual landing vector that leads to a secret docking bay below ground in Argentina. You land and a Navy cadet meets you at your ship with an urgent message, "Republic Intelligence and a Navy Marshal want to see you as soon as possible. They have news about your microtransceiver transmissions. Please follow me."`

Reviewer 2: ```suggestion
			`The Earth authorities give you an unusual landing vector that leads to a secret docking bay below ground in Argentina. You land, and a Navy cadet meets you at your ship with an urgent message: "Republic Intelligence and a Navy Marshal want to see you as soon as possible. They have news about your microtransceiver transmissions. Please follow me."`
```

			`	Giti adds, "I'll take the scanners and data to some researchers and let you know when I have more information. Please go with the cadet. It sounds like you'll finally get some answers."`

> 			`	The cadet takes you to an elevator just outside your ship, down more than three hundred floors to a large meeting room. Within, you see two men and a woman in formal dress. One man has a Republic Intelligence uniform with the name, Agent Kopkow. Another wears a navy full dress uniform that you don't immediately recognize. After a moment of thought, you realize he is a Marshal, the second-highest rank in the Navy, answering only to the Grand Marshal. His uniform is is labeled, Adurath Kaine. The woman is a Navy captain wearing the prominent Parliamentary Medal of Valor. Her name label says, "Captain Anna Wallace."`

Reviewer 2: ```suggestion
			`	The cadet takes you to an elevator just outside your ship, down more than three hundred floors to a large meeting room. Within, you see two men and a woman in formal dress. One man has a Republic Intelligence uniform with the name "Agent Kopkow". Another wears a Navy full dress uniform that you don't immediately recognize. After a moment of thought, you realize he is a Marshal, the second-highest rank in the Navy, answering only to the Grand Marshal. His uniform is is labeled "Adurath Kaine". The woman is a Navy captain wearing the prominent Parliamentary Medal of Valor. Her name label says "Captain Anna Wallace."`
```

			choice

> 				`	(Speak to the agent and marshal.)`

Reviewer 2: ```suggestion
				`	(Speak to the agent and Marshal.)`
```
I assume `Marshal` is a proper noun based on its usage above.

					goto join
				`	(Ruminate first.)`

> 			`	You've had few dealings with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle. Murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament.`

Reviewer 2: ```suggestion
			`	You've had few dealings with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament.`
```

> 			`	There are only five Navy Marshals, and one Grand Marshal; this man's high rank means he reports parliament regularly. People this high in the navy tend to deal more with political issues, rather than combat or strategy, so you wonder why they picked him to talk to you.`

Reviewer 2: ```suggestion
			`	There are only five Navy Marshals, and one Grand Marshal; this man's high rank means he reports to Parliament regularly. People this high in the Navy tend to deal more with political issues rather than combat or strategy, so you wonder why they picked him to talk to you.`
```
Is `Parliament` capitalized? Dunno, but I thought it might need to be. I can edit the suggestion to not include it if that's wrong

			`	The Parliamentary Medal of Valor is only given to one person per year; this captain must have done something especially heroic. She must also have supporters in Parliament to receive the award in the first place. That will give her some freedom to act independently, like Danforth does.`
			`	The three of them are looking at you confused, wondering why you've stopped and stared at them for a half minute.`
			choice
				`	(Stop staring blankly and speak to them.)`
			label join

> 			`	Marshal Kaine introduces himself, "I'm the Marshal of the Third Armada, the guards of the Far North and the uninhabited regions near it. I know you've met one of my admirals, Danforth that is, a good man albeit one who flouts the rules a bit too much." He gives a stern look at Captain Wallace. "Some of my carriers have detected the signals you've found. Actually, we see them quite regularly in the ships of known slave runners. The runners don't always carry slaves; they do other smuggling missions as well."`

Reviewer 2: ```suggestion
			`	Marshal Kaine introduces himself. "I'm the Marshal of the Third Armada, the guards of the Far North and the uninhabited regions near it. I know you've met one of my admirals, Danforth that is, a good man - albeit one who flouts the rules a bit too much." He gives a stern look at Captain Wallace. "Some of my carriers have detected the signals you've found. Actually, we see them quite regularly in the ships of known slave runners. The runners don't always carry slaves; they do other smuggling missions as well."`
```

> 			`	Agent Kopkow speaks his piece next, "Sometimes the runners do carry slaves, and we have found humans or Hai in crashed slaver ships. We returned the Hai to their homes, as you did, but they didn't know Republic Intelligence helped them. Several merchants have done the same. About one in seven we rescued were researchers, an extremely disproportionate amount when you consider the number of researchers in human and Hai space. Clearly they're being targeted."`

Reviewer 2: ```suggestion
			`	Agent Kopkow speaks his piece next. "Sometimes the runners do carry slaves, and we have found humans or Hai in crashed slaver ships. We returned the Hai to their homes, as you did, but they didn't know Republic Intelligence helped them. Several merchants have done the same. About one in seven we rescued were researchers, which is an extremely disproportionate amount when you consider the number of researchers in human and Hai space. Clearly, they're being targeted."`
```

> 			`	Kaine continues, "That, unfortunately, is where we have some bad news. Parliament has a strict policy on pirates and slaver ships: destroy on sight. This is a tradition that goes back long before the era of space travel. Raiding pirate ships is a deadly affair; numerous navy or police will be killed. While it may seem heroic, the casualty rate is unsustainable. The only effective way to discourage piracy is to destroy the ships. If you don't believe me, try catching some yourself and count your dead."`

Reviewer 2: ```suggestion
			`	Kaine continues, "That, unfortunately, is where we have some bad news. Parliament has a strict policy on pirates and slaver ships: destroy on sight. This is a tradition that goes back long before the era of space travel. Raiding pirate ships is a deadly affair; numerous Navy or police will be killed. While it may seem heroic, the casualty rate is unsustainable. The only effective way to discourage piracy is to destroy the ships. If you don't believe me, try catching some yourself and count your dead."`
```
I like this little nod to how unsustainable the process of capturing ships would actually be.

			`	Captain Wallace concludes, "That means many slaves are killed, including some Hai researchers."`

> 			`	Kaine says, "We informed the Hai government while you were away and they've responded by moving fleets towards the wormhole; we're doing the same. By the time you arrive, there will be large fleets on both sides of the wormhole. This could quickly lead to war. Please take the message back to Teeneep and the Hai victims with our condolences. Captain, please see <first> <last> back to the <ship>."`

Reviewer 2: ```suggestion
			`	Kaine says, "We informed the Hai government while you were away, and they've responded by moving fleets towards the wormhole; we're doing the same. By the time you arrive, there will be large fleets on both sides of the wormhole. This could quickly lead to war. Please take the message back to Teeneep and the Hai victims with our condolences. Captain, please see <first> <last> back to the <ship>."`
```

			choice
				`	(Return to your ship.)`
			`	Along the way, Captain Wallace tells you stories of Navy heroism and brutality towards slaver vessels. Apparently many officers like her disagree with the Navy's policy. They want to rescue slaves whenever possible, regardless of risk or orders.`
			`	She was awarded for personally leading a charge deep into pirate space to rescue a group of over a hundred high-school students that were captured on a field trip to Shroud. That led to her court-martial and conviction for disobeying a direct order. She was pardoned by the Prime Minister upon receiving the Parliamentary Award for Valor.`

> 			`	When you reach your ship, she hands you a business card, "Please have the Hai victims talk to this man. He's part of Syndicate Internal Affairs and may be able to help. He and I... share common goals."`

Reviewer 2: ```suggestion
			`	When you reach your ship, she hands you a business card. "Please have the Hai victims talk to this man. He's part of Syndicate Internal Affairs and may be able to help. He and I... share common goals."`
```

# Copyright (c) 2020 Nomadic Volcano
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

########################################################################

# Part 4: Search Korath and human space for the missing researchers

mission "Hai Rescue [A01] Visit Darkwaste"
	landing
	name "Missing Researchers"
	description "Fly to <destination> to visit the Hai you rescued."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
		not attributes "uninhabited"
		not planet "Darkwaste"
	destination Darkwaste
	to offer
		has "event: hai rescue: teeneep message"
		has "Hai Building Mountaintop [3] Furnishings: done"
		or
			has "Pirate Troubles [4]: done"

> 			has "Pirate Troubles [4]: declined"

Author: In this case the pirate troubles set gets given an additional condition in its `to offer` which means during the process of finding and returning Hai, and then building Mountaintop you *will* trigger the pirate troubles mission somewhere.

This is because it cannot narratively happen after this, and I'm doing my best not to lock out existing content accidentally for people who haven't discovered the Hai.

But it doesn't care if you do it or decline it, just that you've had the opportunity to do either.

	on offer
		conversation

> 			`You receive an emergency communication from a Hai woman named Teeneep. The message is brief and cryptic, but you deduce that it refers to the Hai you've rescued from human space in the past. They need to talk to you again as soon as possible. There is a set of landing coordinates on the uninhabited Hai planet, <destination>.`

Author: Just as a note for future, I would like to do this, but not without #5440 
It's on the "to refine later" list.

	system "Zaurak"
		add fleet "Smuggler (Hai Traffickers)" 7000
	system "Sheratan"
		add fleet "Smuggler (Hai Traffickers)" 7000
	system "Ruchbah"
		add fleet "Smuggler (Hai Traffickers)" 7000
	system "Alpheratz"
		add fleet "Smuggler (Hai Traffickers)" 7000
	system "Achernar"
		add fleet "Smuggler (Hai Traffickers)" 7000
	system "Al Dhanab"
		add fleet "Smuggler (Hai Traffickers)" 7000
	system "Mirach"
		add fleet "Smuggler (Hai Traffickers)" 7000
	
	# Staging areas for transfers
	system "Alcyone"
		add fleet "Smuggler (Hai Traffickers)" 2500
	system "Almaaz"
		add fleet "Smuggler (Hai Traffickers)" 2500
	system "Arneb"
		add fleet "Smuggler (Hai Traffickers)" 2500
	system "Alnilam"
		add fleet "Smuggler (Hai Traffickers)" 2500

	# Final holding area before Devil-Run
	system "Almach"
		add fleet "Smuggler (Hai Traffickers)" 1000
	

mission "Hai Rescue [A04] Tarazed Analysis"
	landing
	name "Missing Researchers"
	description `Find and board a ship that has microtransceivers. Your display will identify such ships as government "Smuggler." Try the uninhabited systems and pirate systems near Ultima Thule. Head to <destination> afterwards.`
	source Wayfarer
	destination Earth

> 	clearance `Curious about your story, the authorities let you land.`

Author: I think it's more the principle of the thing, you should have clearance either way. However, on consideration the text associated with this is... less than great.

			`	He's a little groggy, but manages to ask, "Yes?" You explain about the microtransceivers, the Quarg, and the kidnappings. You avoid mentioning that some of the people are Hai, just in case he's never heard of them. He seems to awaken immediately. "You found a lead on my wife's disappearance? Yes, I think I can help you. Let's go to my laboratory at once."`
			choice
				`	(Head to the laboratory.)`
			`	After another series of shuttles and trains, you reach what looks like a spaceport. From the extra equipment and unusual ship designs, you realize it is a research shipyard. Goraz's laboratory is an entire docking bay with a modified Blackbird. Despite its size, it is conspicuously absent any other personnel. He takes you on board and, noting your attention to your surroundings, explains that he has few assistants that are typically off-world most of the time as he walks you to the communications room.`

> 			`	"Here, let me take a look at what you've brought me." He takes the data card you got from the Quarg and plugs it into the communicator. After several minutes of typing and muttering incoherently to himself he pauses as a simulation plays on his screen. You're not sure exactly what you're looking at, but when it finishes he lets out a long slow breath and comments to himself in a low voice. "Bastijn Biesterveld, you absolute legend. He was right - it's the interference."`

Author: They're Dutch

> 			`	"Here, let me take a look at what you've brought me." He takes the data card you got from the Quarg and plugs it into the communicator. After several minutes of typing and muttering incoherently to himself he pauses as a simulation plays on his screen. You're not sure exactly what you're looking at, but when it finishes he lets out a long slow breath and comments to himself in a low voice. "Bastijn Biesterveld, you absolute legend. He was right - it's the interference."`

Reviewer 7: ~~Even worse.~~

				accept

mission "Hai Rescue [A05] Boarded Smuggler"
	name "Missing Researchers"
	description "You have the information Giti Gilani will need. It is time to visit her on <destination>."
	boarding
	destination Earth
	to offer
		has "Hai Rescue [A04] Tarazed Analysis: active"
	source
		government "Smuggler (Hai Trafficker)"
	on offer
		conversation
			`As you approach the airlock of the smuggler ship <origin>, the door opens and a Hai rushes through pointing a gun back towards a human in the ship. The human and Hai shoot at the same time, killing each other. In his death throes, the Hai manages to toss a keycard in your direction. It is damaged beyond usefulness, but you can see a pair of logos on it. One looks like an angry cat in a circle, and the other looks demonic.`
			`	Soon after, you hear explosions in the ship and see an escape pod float away. It seems the smugglers set the self-destruct and signaled for help. The pod has no microtransceiver transmissions, so it must be one of the smugglers. Apparently the smugglers care more about not being caught than about finishing their mission.`

> 			`	You begin to set a course for <planet> where you can finally get some answers, but multiple hyperspace windows suddenly begin to form. Your scanners indicate hostile ships are entering the system. You prepare for battle.`

Author: Lol, the things that manage to slip past us all.
```suggestion
			`	You begin to set a course for <planet> where you can finally get some answers, but multiple signals start appearing on your scanners as hostile ships enter the system. You prepare for battle.`
```

		conversation
			`After landing, you search for the "Republic Historical Space-Flight Museum," and find there are seventeen museums by that name on Earth.  Only one has a research library, so you contact the library. Giti Gilani no longer works there; she's moved to a library in the Navy Intelligence Observatory in Moscow.`
			`	The Navy observatory was easier to find; when they hear your name, they put you in contact with Giti immediately. She is excited to see the two symbols you found and asks you to visit her. It takes three shuttle journeys and one high-speed rail trip, but you reach the building in under three hours.`
			choice
				`	(Enter the building.)`
			`	You're ushered to an elevator and given a room number: 3017A19 - the 301st floor, section 7A, room 19. It's amazing how large Earth's megabuildings have become. Some planets have less population than one of these buildings.`

> 			`	Giti is even more excited in-person than she was by holocommunicator. She's already researched the Syndicate-like symbol, but needed more detail about the demonic one. You also show her the information about the microtransceivers.`

Reviewer 7: Syndicate-like symbol? This is the first time the Syndicate have been mentioned.

			`	"But you didn't come here for a theological history disquisition. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!`
			`	"The circle around the symbol looks almost like a wormhole. A red wormhole. I heard about one in Terminus near The Deep. Apparently nobody is able to enter one without their ship being torn to pieces. This symbol suggests there's another one in the middle of Syndicate space.`
			`	"Fly through the wormhole at the coordinates on that card, if you can, come back and tell me what you see. In the meantime, I'll ask other researchers about the microtransceivers."`
			choice
				`	(Thank her and return to your ship.)`

> 			`	Upon returning to your ship, you ponder Giti's findings. The coordinates are for a system in Syndicate space that is not reachable by hyperlane, so you'll need a Jump Drive. The red wormhole requires another key.`

Reviewer 7: ```suggestion
			`	Upon returning to your ship, you ponder Giti's findings. The coordinates are for a system in Syndicate space that is not reachable by hyperlane, so you'll need a jump drive. The red wormhole requires another key.`
```
This "other key" is never described. The player could theoretically be stuck here for a significant amount of time if they never figure out they need a keystone.

		conversation
			`After a long series of shuttles, high-speed rails, and a several-minute-long elevator ride, you reach Giti's office.`
			`	As soon as she sees you she asks, "What did you find out?" You explain your journey to her.`
			`	She responds, "Wait... I... lemme check..." followed by some typing on a nearby terminal. "Yes, you're right. In the late 21st century, there was an effort to map stars in that intergalactic region. By all rights these stars should not have been so visible, but some strange anomaly made that part of the sky clearer for a while. The strangest thing we spotted was not a star; it was some kind of object with unknown properties that confused our scanners. Even today I'm not sure we'd make sense of it, but the anomaly passed, and we couldn't even make an attempt now."`
			`	"If the Syndicate criminals and kidnappers are through the wormhole, that's the system they'll be in."`
			`	"Let me contact some people. Hold on..." Her voice trails off as she begins typing on a nearby terminal.`
			choice
				`	(Wait.)`
					goto wait

> 				`	(Interrupt her.)`

Reviewer 7: So why are all the choices suddenly action choices?

			`	"What? I... Aaa! Don't startle me like that. I just need a minute. Hold on..."`
			label wait
			`	"Okay. I've arranged for some advanced scanning equipment that can see past most types of radiation and a few hundred meters of solid rock. I'll have to come with you on this trip."`

> 			`	"Oh! I almost forgot to tell you the good news. I found some people who know about the microtransceiver transmissions. Navy carriers have sensitive communication arrays that can pick up the transmissions. The Navy is combing their communication logs and will have some answers when we return."`

Reviewer 7: ```suggestion
			`	"Oh! I almost forgot to tell you the good news. I found some people who know about the microtransceiver transmissions. Navy Carriers have sensitive communication arrays that can pick up the transmissions. The Navy is combing their communication logs and will have some answers when we return."`
```

		conversation
			`Giti turns on the scanning equipment.`
			`	"Okay, let's do the planet first..." Her voice trails off. "Wow! That's odd. We'll need a scanner about ten thousand times more powerful to scan that planet."`
			`	 "I can see a bit of the surface though. The planet is odd in every way. It is completely inhospitable to all known forms of life. There's a huge network of caves and caverns coated with a crystal that is radiating... something the computer can't identify... and deuterium. The denizens of this system siphon the deuterium to make hyperspace fuel. Also, the planet has about a thousand times the mass it should, but somehow the gravity is around Earth normal."`
			`	"Now the moon... hmmm... okay. It's just an ordinary chunk of rock, but the planet's radiation is disturbing the sensors. I can only see a few dozen meters below ground."`

> 			`	"I found your missing researchers. They're in a facility beneath the moon's surface. There are three different species there: Hai, human, and some kind of reptile." You realize she probably means a Korath. "No Alphas though. I can only see the uppermost part of the facility. They have hydroponics, big cargo bays, docking bays for ships, and some spacious living areas. I detect 1,387 life signs spanning all age ranges, and that's just in the upper three floors of the facility."`

Reviewer 7: Why bring up the Alphas? How in the world are these scanners capable of identifying not only the exact number of people in a building, but also their species and ages?

			`	"This isn't a base; it's a colony."`

> 			`	You notice capital ships heading your way and about two dozen more preparing to leave the planet. It's time to go.`

Reviewer 7: This kinda comes out of nowhere.

			`	Agent Kopkow speaks his piece next. "Sometimes the runners do carry slaves, and we have found humans or Hai in crashed slaver ships. We returned the Hai to their homes, as you did, but they didn't know Republic Intelligence helped them. Several merchants have done the same. About one in seven we rescued were researchers, which is an extremely disproportionate amount when you consider the number of researchers in human and Hai space. Clearly, they're being targeted."`
			`	Kaine continues, "That, unfortunately, is where we have some bad news. Parliament has a strict policy on pirates and slaver ships: destroy on sight. This is a tradition that goes back long before the era of space travel. Raiding pirate ships is a deadly affair; numerous Navy or police will be killed. While it may seem heroic, the casualty rate is unsustainable. The only effective way to discourage piracy is to destroy the ships. If you don't believe me, try catching some yourself and count your dead."`
			`	Captain Wallace concludes, "That means many slaves are killed, including some Hai researchers."`

> 			`	Kaine says, "We informed the Hai government while you were away, and they've responded by moving fleets towards the wormhole; we're doing the same. By the time you arrive, there will be large fleets on both sides of the wormhole. This could quickly lead to war. Please take the message back to Teeneep and the Hai victims with our condolences. Captain, please see <first> <last> back to the <ship>."`

Reviewer 7: So only now has this trafficking ring that's been apparently going on for decades or even centuries that everyone's been blasting out of the sky the entire time led to a potential conflict between the Republic and Hai? Seems like this comes out of nowhere.

Reviewer 8: The Hai apparently just didn't know about it? And the Republic have decided that just now is the best timme to tell them about all the Hai kidnapping victims they've been murderising?
I guess the Republic is as stupid as the Hai leadership.

# Copyright (c) 2020 Nomadic Volcano
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

########################################################################

# Part 4: Search Korath and human space for the missing researchers

#Timer trigger runs off Reveal 2 events, and ensures a minimum time between events if you do Reveal 3 events very quickly.
mission "Hai Rescue [A00] Darkwaste Timer"
	landing
	invisible
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
		not attributes "uninhabited"
		not planet Darkwaste
	destination Darkwaste
	to offer
		"hai slave prereq" >= 3
		"combat rating" > 3000
		has "Hai Leaks Response 6: done"
	on offer

> 		event "hai rescue: teeneep message" 20 30

Author: If you happen to have discovered the two which you can discover accidentally before the plot, ~10 days.

The timer is purely to add some distance if you happen to be ahead of the game so you can get the experience of continuing to hunt for other mission possibilities.

		fail

event "hai rescue: teeneep message"

mission "Hai Rescue [A01] Visit Darkwaste"
	landing
	name "Missing Researchers"
	description "Fly to <destination> to visit the Hai you rescued."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
		not attributes "uninhabited"
		not planet "Darkwaste"
	destination Darkwaste
	to offer
		has "event: hai rescue: teeneep message"
		has "Hai Building Mountaintop [3] Furnishings: done"
		or
			has "Pirate Troubles [4]: done"
			has "Pirate Troubles [4]: declined"
	on offer
		conversation
			`You receive an emergency communication from a Hai woman named Teeneep. The message is brief and cryptic, but you deduce that it refers to the Hai you've rescued from human space in the past. They need to talk to you again as soon as possible. There is a set of landing coordinates on the uninhabited Hai planet, <destination>.`
				accept

mission "Hai Rescue [A02] Microtransceiver Identification"
	landing
	name "Missing Researchers"
	description "Fly to <destination> to ask the Quarg about the microtransceivers."
	source Darkwaste

> 	destination "Kuwaru Efreti"

Author: Well, on the plus side, there should be a lot more things to go out and discover than just those soon enough.
This whole Kuwaru Efreti bit may well get redone from scratch though.

> 	destination "Kuwaru Efreti"

Reviewer 3: Especially if it's slated for larger rework anyway, I agree it would probably be best to skip sending us to the Efreti.

The Hai or Human Quarg should suffice (if they're even still included at all).

			`	"This isn't a base; it's a colony."`
			`	You notice capital ships heading your way and about two dozen more preparing to leave the planet. It's time to go.`
	on complete
		payment 775000
	npc
		government "Pirate (Devil-Run Gang)"
		system "Devil-Hide"
		fleet "Devil-Run Warships" 2
		personality nemesis harvests plunders disables

mission "Hai Rescue [A09] Slaver Ships"
	landing
	name "Missing Researchers"
	description "Report back to Teeneep on <destination>. Payment is <payment>."
	source "Earth"
	destination "Darkwaste"
	to offer
		has "Hai Rescue [A08] Scanner: done"
	on offer
		conversation
			`The Earth authorities give you an unusual landing vector that leads to a secret docking bay below ground in Argentina. You land, and a Navy cadet meets you at your ship with an urgent message: "Republic Intelligence and a Navy Marshal want to see you as soon as possible. They have news about your microtransceiver transmissions. Please follow me."`
			`	Giti adds, "I'll take the scanners and data to some researchers and let you know when I have more information. Please go with the cadet. It sounds like you'll finally get some answers."`

> 			`	The cadet takes you to an elevator just outside your ship, down more than three hundred floors to a large meeting room. Within, you see two men and a woman in formal dress. One man has a Republic Intelligence uniform with the name "Agent Kopkow". Another wears a Navy full dress uniform that you don't immediately recognize. After a moment of thought, you realize he is a Marshal, the second-highest rank in the Navy, answering only to the Grand Marshal. His uniform is labeled "Adurath Kaine". The woman is a Navy captain wearing the prominent Parliamentary Medal of Valor. Her name label says "Captain Anna Wallace."`

Reviewer 8: Marshal? And... Grand Marshal? That's what we're going with? Not a fan.

			choice
				`	(Speak to the agent and Marshal.)`
					goto join
				`	(Ruminate first.)`

> 			`	You've had few dealings with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament.`

Reviewer 8: Thanks for the exposition?

			`	The Parliamentary Medal of Valor is only given to one person per year; this captain must have done something especially heroic. She must also have supporters in Parliament to receive the award in the first place. That will give her some freedom to act independently, like Danforth does.`
			`	The three of them are looking at you confused, wondering why you've stopped and stared at them for a half minute.`
			choice
				`	(Stop staring blankly and speak to them.)`
			label join
			`	Marshal Kaine introduces himself. "I'm the Marshal of the Third Armada, the guards of the Far North and the uninhabited regions near it. I know you've met one of my admirals, Danforth that is, a good man - albeit one who flouts the rules a bit too much." He gives a stern look at Captain Wallace. "Some of my carriers have detected the signals you've found. Actually, we see them quite regularly in the ships of known slave runners. The runners don't always carry slaves; they do other smuggling missions as well."`

> 			`	Agent Kopkow speaks his piece next. "Sometimes the runners do carry slaves, and we have found humans or Hai in crashed slaver ships. We returned the Hai to their homes, as you did, but they didn't know Republic Intelligence helped them. Several merchants have done the same. About one in seven we rescued were researchers, which is an extremely disproportionate amount when you consider the number of researchers in human and Hai space. Clearly, they're being targeted."`

Reviewer 8: F*** me. Did no one tell the Hai authorities about these kdinapped Hai found in human space???
*rages*

# Copyright (c) 2020 Nomadic Volcano
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer 9: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.
```

				`	"How can I help you?"`
					goto help
			label uninhabited

> 			`	Teeneep looks at you as if you were daft. "The kidnappings happen on Hai and human worlds. The kidnappers have no reason to search an uninhabited planet for potential victims. This gives us a safe place to hide any critical people or information. Plus once we've removed enough of the microtransceivers they no longer function effectively. This place gives us plenty of space to keep samples and people far enough apart for safety without the obvious precautions that would be necessary somewhere more populated."`

Reviewer 2: ```suggestion
			`	Teeneep looks at you as if you were daft. "The kidnappings happen on Hai and human worlds. The kidnappers have no reason to search an uninhabited planet for potential victims. This gives us a safe place to hide any critical people or information. Plus, once we've removed enough of the microtransceivers, they no longer function effectively. We have plenty of space here to keep samples and people far enough apart for safety without the obvious precautions that would be necessary somewhere more populated."`
```

		has "ZZ Hai Rescue [A03] Microtransceiver Identification: done"
	to complete
		has "Hai Rescue [A05] Boarded Smuggler: offered"
	on visit
		dialog `You have not yet boarded a ship to learn anything about the smugglers. You will need to do this to continue.`
	on offer
		event "devil-run smugglers visible"
		log "People" "Goraz Zoltof" `Goraz is a communications researcher from Tarazed Shipyards. His wife went missing some time ago, and he hopes his research will lead to her rescue.`
		log "Factions" "Smugglers" `Some pirate or merchant ships (identified by the computer as "Smuggler") carry slaves with special microtransceivers. They are probably carrying human or Hai researchers.`
		log `Goraz Zoltof used the galactic network of television transmitters to scan for evidence of smuggling ships carrying kidnapping victims with microtransceivers. He detected hints of transmissions in the uninhabited space between the Far North and the Syndicate, and also near pirate systems.`
		conversation

> 			`You read Teeneep's contact information for Tarazed Shipyards, and it talks about a man named Goraz Zoltof, who works in their communications research laboratory. There's an address and detailed information about the man whose wife went missing under mysterious circumstances some time ago. After a series of shuttles and trains, you reach his house and knock on the door.`

Reviewer 2: ```suggestion
			`You read Teeneep's contact information for Tarazed Shipyards, and it talks about a man named Goraz Zoltof who works in their communications research laboratory. There's an address and detailed information about the man, whose wife went missing under mysterious circumstances some time ago. After a series of shuttles and trains, you reach his house and knock on the door.`
```

			`	He's a little groggy, but manages to ask, "Yes?" You explain about the microtransceivers, the Quarg, and the kidnappings. You avoid mentioning that some of the people are Hai, just in case he's never heard of them. He seems to awaken immediately. "You found a lead on my wife's disappearance? Yes, I think I can help you. Let's go to my laboratory at once."`
			choice
				`	(Head to the laboratory.)`
			`	After another series of shuttles and trains, you reach what looks like a spaceport. From the extra equipment and unusual ship designs, you realize it is a research shipyard. Goraz's laboratory is an entire docking bay with a modified Blackbird. Despite its size, it is conspicuously absent any other personnel. He takes you on board and, noting your attention to your surroundings, explains that he has few assistants that are typically off-world most of the time as he walks you to the communications room.`

> 			`	"Here, let me take a look at what you've brought me." He takes the data card you got from the Quarg and plugs it into the communicator. After several minutes of typing and muttering incoherently to himself he pauses as a simulation plays on his screen. You're not sure exactly what you're looking at but when it finishes he lets out a long slow breath and comments to himself in a low voice. "Bastijn Biesterveld you absolute legend. He was right, it's the interference."`

Reviewer 2: ```suggestion
			`	"Here, let me take a look at what you've brought me." He takes the data card you got from the Quarg and plugs it into the communicator. After several minutes of typing and muttering incoherently to himself he pauses as a simulation plays on his screen. You're not sure exactly what you're looking at, but when it finishes he lets out a long slow breath and comments to himself in a low voice. "Bastijn Biesterveld, you absolute legend. He was right - it's the interference."`
```

> 			`	He turns back to you and explains. "Normally interference isn't something that's logged, but when you're in civilized space with hyperspace relays as your ship jumps it does a final handshake with the local relay station. That station sends that handshake signature to the destination station so it knows exactly who you are to be able to reconnect communications at the other end. It's a fully automatic process and it's independent of ship ID's, it's just buried in there as a check from back in the days when hyperdrives would occasionally explode and we'd need to work out who went missing. It gets logged and automatically overwritten every couple days, and someone would probably have gotten rid of it if it wasn't so deeply baked into the way the system works. If you know when a ship jumped and where it jumped to, you can use it to confirm it didn't blow up; and if there's something wrong with the communications equipment you can use the log info to identify if a specific ship is the same ship talking to you later, but neither of those are particularly useful since the same information can be acquired more easily and in much higher quality through other means."`

Reviewer 2: ```suggestion
			`	He turns back to you and explains. "Normally interference isn't something that's logged, but when you're in civilized space with hyperspace relays, your ship does a final handshake with the local relay station as it jumps. That station sends that handshake signature to the destination station so it knows exactly who you are to be able to reconnect communications at the other end. It's a fully automatic process, and it's independent of ship IDs. It's just buried in there as a check from back in the days when hyperdrives would occasionally explode and we'd need to work out who went missing. It gets logged and automatically overwritten every couple days, and someone would probably have gotten rid of it if it wasn't so deeply baked into the way the system works. If you know when a ship jumped and where it jumped to, you can use it to confirm it didn't blow up; and if there's something wrong with the communications equipment you can use the log info to identify if a specific ship is the same ship talking to you later, but neither of those are particularly useful since the same information can be acquired more easily and in much higher quality through other means."`
```

> 			`	He stops and looks sheepish for a moment. "Sorry, that was probably way more explanation than you needed. There's a small group of us comms people that have lost people or know people who have, that the authorities can't help. We're scattered all over and we've been looking at this ourselves. Together we identified some ships we thought were suspicious but we've not been able to pin down anything concrete. Bastijn was convinced there was a pattern of interference signatures from the logs; he made the analysis program but it was never definitively distinct from the noise. Now though, well what you've given me is a match!"`

Reviewer 2: ```suggestion
			`	He pauses. "Sorry, that was probably way more explanation than you needed. There's a small group of us comms people that have lost people, or know people who have, that the authorities can't help. We're scattered all over and we've been looking at this ourselves. Together we identified some ships we thought were suspicious, but we've not been able to pin down anything concrete. Bastijn was convinced there was a pattern of interference signatures from the logs; he made the analysis program, but it was never definitively distinct from the noise. Now though, well, what you've given me is a match!"`
```

Author: "pauses sheepishly" perhaps?

Reviewer 2: More of a personal preference thing, I think you use `sheepish` a bit much but that's just me :P
I'll let you decide:
```suggestion
			`	He pauses, realizing he's been rambling. "Sorry, that was probably way more explanation than you needed. There's a small group of us comms people that have lost people, or know people who have, that the authorities can't help. We're scattered all over and we've been looking at this ourselves. Together we identified some ships we thought were suspicious, but we've not been able to pin down anything concrete. Bastijn was convinced there was a pattern of interference signatures from the logs; he made the analysis program, but it was never definitively distinct from the noise. Now though, well, what you've given me is a match!"`
```
```suggestion
			`	He pauses sheepishly. "Sorry, that was probably way more explanation than you needed. There's a small group of us comms people that have lost people, or know people who have, that the authorities can't help. We're scattered all over and we've been looking at this ourselves. Together we identified some ships we thought were suspicious, but we've not been able to pin down anything concrete. Bastijn was convinced there was a pattern of interference signatures from the logs; he made the analysis program, but it was never definitively distinct from the noise. Now though, well, what you've given me is a match!"`
```

Author: It's a good thing I'm not a New Zealander or there'd be a good joke in there. I'll go with the rambling one, I probably do overuse it.

> 			`	He casts around for a moment and then plugs in a freshly unpacked data card into his console. "Gimme a minute I can do something for you now. Just uh, sit down for ten minutes and I'll be back with you."`

Reviewer 2: ```suggestion
			`	He casts around for a moment and then plugs in a freshly unpacked data card into his console. "Give me a little time: I can do something for you now. I'll be back with you in ten minutes."`
```
The dialogue style suddenly changed for this line when compared to what's before it.
Also, what's `casts` mean? Don't quite understand how it's being used here.

Author: "To look around for something" - but it connotes looking in a manner where you're mostly looking in your mind for where you expect to find a thing as opposed to physically using your eyes to identify something.
e.g. when you're mentally rifling through the draws of your desk to recall which draw you put something in, or scanning a messy table in the full knowledge that what you're looking for is probably covered by something but with visual context clues you can retrace your steps mentally to recall where you left it.

https://dictionary.cambridge.org/dictionary/english/cast-around

In this context it implies he's either a) working out where he left his fresh drives to get one, b) thinking where he might have an unused data port to plug something in to - assuming he's using most of his ports semi-permanently, or c) both.

Reviewer 2: Gotcha, thanks! I'm good with it here then.

			choice
				`	(Sit down.)`
				`	(Stand around idly.)`

> 			`	After some time, in which he looks like he's installing a series of programs onto it, he returns your original data card and the second one, now loaded up with whatever he was putting on it. "Load this new card into one of your communicator's extra data storage slots, preferably the first one. This new card will run maintenance level processes with the communicator to download the logs from the local station when you enter a system and match the log signatures for ships that are in-system and code up extra info for their ID's. Functionally, they'll show up as 'Smuggler' on your display. You'll have to hunt for them though, cause you'll have to be in the same system as them for any of this to work.`

Reviewer 2: ```suggestion
			`	After some time, in which he looks like he's installing a series of programs, he returns your original data card and the second one, now loaded up with whatever he was putting on it. "Load this new card into one of your communicator's extra data storage slots, preferably the first one. This new card will run maintenance level processes with the communicator to download the logs from the local station when you enter a system, match the log signatures for ships that are in-system, and code up extra info for their ID's. Functionally, they'll show up as 'Smuggler' on your display. You'll have to hunt for them though, cause you'll have to be in the same system as them for any of this to work.`
```

> 			`	"Also uh, maybe keep this quiet. This is..." He clears his throat conspicuously. "Somewhat less than strictly legal, though there's no penalty attached to accessing the data at the level this will do for you. You should focus your search on the uninhabited areas between the Far North and Syndicate space, plus any nearby pirate systems. That's where we've noticed the most suspicious activity."`

Reviewer 2: ```suggestion
			`	"Also... maybe keep this quiet. This is"-he pauses and clears his throat conspicuously-"somewhat less than strictly legal, though there's no penalty attached to accessing the data at the level this card will use. You should focus your search on the uninhabited areas between the Far North and Syndicate space, plus any nearby pirate systems. That's where we've noticed the most suspicious activity."`
```

Author: I do not like this application of grammar, it disagrees with a lot of what I've been taught. Gotta ask if this structure is a common americanism again?

Reviewer 2: Not sure if this is predominantly American or not. Here's a couple references I found/consulted on punctuating interruptions to dialogue:
https://theeditorsblog.net/2018/04/08/when-a-comma-isnt-enough/
https://www.touchstone-editing.com/2017/10/mini-lesson-punctuating-interrupted.html
Though the way you have it might be right. I do prefer my suggestion though, because it feels like including a whole sentence between `This is...` and `somewhat less than` makes that break between the complete thought of "this is less than legal" longer than it should be, if that makes sense.

> 			`	All business and technical explanation now out of the way you watch as the reality of what's happening seems to suddenly sink in and his demeanor changes suddenly as his eyes shine with faint hope, "And please, if you do find my wife Janitza... please bring her home."`

Reviewer 2: ```suggestion
			`	All business and technical explanation now out of the way, you watch as the reality of what's happening seems to suddenly sink in, and his demeanor changes suddenly as his eyes shine with faint hope, "And please, if you do find my wife Janitza... please bring her home."`
```
